docs/ipr/scripting/RNA_Expression_Similarity.ipynb

```diff
@@ -0,0 +1,4625 @@
+              "COAD                         545\n",
+              "DLBC                          48\n",
+              "ESCA                         204\n",
+              "GBM                          602\n",
+              "HNSC                         604\n",
+              "KICH                          91\n",
+              "KIRC                         944\n",
+              "KIRP                         352\n",
+              "LAML                         200\n",
+              "LGG                          529\n",
+              "LIHC                         438\n",
+              "LUAD                         641\n",
+              "LUSC                         623\n",
+              "MESO                          87\n",
+              "OV                           604\n",
+              "PAAD                         196\n",
+              "PCPG                         187\n",
+              "PRAD                         566\n",
+              "READ                         183\n",
+              "SARC                         271\n",
+              "SKCM                         479\n",
+              "STAD                         511\n",
+              "TGCT                         139\n",
+              "THCA                         580\n",
+              "THYM                         126\n",
+              "UCEC                         583\n",
+              "UCS                           57\n",
+              "UVM                           80"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 5
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "yAOO45wmiIYy"
+      },
+      "source": [
+        "Now we will select a couple of cohorts\n",
+        "- CHOL\n",
+        "- LIHC\n",
+        "- SKCM"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "cLUrlb_0iUIz",
+        "outputId": "c7731f41-47a0-4016-d83f-b7c99aa60d8f"
+      },
+      "source": [
+        "columns_to_keep = set(['sample'] + meta_df[meta_df['cancer type abbreviation'].isin({'CHOL','SKCM', 'LIHC'})]['sample'].tolist())\n",
+        "len(columns_to_keep)"
+      ],
+      "execution_count": 6,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "963"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 6
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_1_wpp_RR1Lm"
+      },
+      "source": [
+        "Now we are going to read in the data matrix. This is a lot of data and so this process is slow. This will still probably take a few minutes. In practice we likely store this in a database to improvement performance"
```

Comment:
change 'we likely' to 'we are likely to'

Reply:
Also this is the second time this sentence has appeared - remove it here or above